mailpoet/tests/acceptance/Subscribers/PageViewTrackingCookieCest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use AcceptanceTester;
+use MailPoet\Test\DataFactories\Subscriber;
+use PHPUnit\Framework\Assert;
+
+class PageViewTrackingCookieCest {
+  private const PAGE_VIEW_COOKIE_NAME = 'mailpoet_page_view';
+
+  public function setCookieForSubscriberStoredInCookie(AcceptanceTester $i) {
+    $i->wantTo('Check page view cookie is set for subscriber saved in cookie');
+
+    $email = 'test-subscriber@example.com';
+    $subscriber = (new Subscriber())->withEmail($email)->create();
+
+    $i->amOnPage('/');
+    $i->cantSeeCookie(self::PAGE_VIEW_COOKIE_NAME);
+    $i->setCookie('mailpoet_subscriber', json_encode(['subscriber_id' => $subscriber->getId()]));
```

Comment:
Is this needed? I mean... will the page view cookie be set even without subscriber cookie?

Reply:
I implemented the cookie to set only if we can detect a subscriber either from a cookie or from logged-in WP User. So that's why I needed to set the subscriber cookie in the test. Do you think we should set always set the cookie even though there is no subscriber to track?

Reply:
I see, makes sense, we don't need the cookie for anything else at the moment.